api/v3/Case/Getwebforms.php

```diff
@@ -47,9 +47,12 @@ function civicrm_api3_case_getwebforms(array $params) {
     return $out;
   }
 
+  $sort = htmlspecialchars($params["options"]["sort"]) ?: "title";
```

Comment:
this will throw a notice if these keys are not provieded, better to use the null coalesce operator  => `??`

Reply:
yes, thanks chief